src/app/rootReducer.ts

```diff
@@ -1,8 +1,10 @@
 import {combineReducers} from "@reduxjs/toolkit";
 import registerReducer from "../containers/Register/RegisterSlice";
+import loginReducer from "../containers/Login/LoginSlice";
 
 const rootReducer = combineReducers({
     register: registerReducer,
+    login: loginReducer,
```

Comment:
maybe it's better that we rename to `user` reducer and actions only and put there everything